### PR TITLE
Documentation pass

### PR DESCRIPTION
**Pull request by the-emerald**
Adds some documentation for functions and constants in the program.

### CODE REVIEW

File: src/gauges.rs [hunk 1]
```diff
@@ -36,6 +37,7 @@ pub struct PrometheusGauges {
 }
 
 impl PrometheusGauges {
+    /// Make new set of gauges.
```
Comment (vkomenda) [inline src/gauges.rs]: Should we have those verbs in third person in function descriptions?

**Review by vkomenda — APPROVED** [review]
A minor nitpick.

File: src/gauges.rs (hunk 1 above)
Comment (the-emerald) [reply src/gauges.rs]: Aha, yep. I'll fix that!

Comment (the-emerald) [reply src/gauges.rs]: Fixed in 0904cb3.

**Pull request merged**